Signal/src/ViewControllers/ConversationView/Cells/OWSMessageTextView.m

```diff
@@ -61,6 +69,61 @@ - (BOOL)pointInside:(CGPoint)point withEvent:(UIEvent *_Nullable)event
     return result;
 }
 
+- (void)setText:(nullable NSString *)text
+{
+    if ([NSObject isNullableObject:text equalTo:self.text]) {
+        return;
+    }
+    [super setText:text];
+    self.cachedSize = nil;
+}
+
+- (void)setAttributedText:(nullable NSAttributedString *)attributedText
+{
+    if ([NSObject isNullableObject:attributedText equalTo:self.attributedText]) {
+        return;
+    }
+    [super setAttributedText:attributedText];
+    self.cachedSize = nil;
+}
+
+- (void)setTextColor:(nullable UIColor *)textColor
+{
+    if ([NSObject isNullableObject:textColor equalTo:self.textColor]) {
+        return;
+    }
+    [super setTextColor:textColor];
+    self.cachedSize = nil;
```

Comment:
Text color changes the size?

Reply:
Yeah, we can skip that here.  Done, and added a comment.  Note that we still want this method; setting text color is not free.